### PR TITLE
Di has made first commit

### PR DESCRIPTION
**Pull request by pea586**
### Description
[Describe what this change achieves]
It is my first commit, OSCI.
 
### Issues Resolved
[List any issues this PR will resolve]
Change folder to OSCI-2023

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/dizhang.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Di Zhang
+short_name: D
+photo: '/assets/media/authors/di.jpg'
```
Comment (joshuarrrr) [inline _authors/osci_2023/dizhang.markdown:4]: @pea586 For now, let's remove the photo

```suggestion
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
Remove the photo - otherwise, looks good.

File: _authors/osci_2023/dizhang.markdown (hunk 1 above)
Comment (pea586) [reply _authors/osci_2023/dizhang.markdown:4]: Sure, I have deleted it and pushed again.

**Review by dtaivpp — APPROVED** [review]
Approved!

**Pull request merged**